tests/settings.py

```diff
@@ -117,7 +117,7 @@
         'rest_framework.authentication.SessionAuthentication',
     ),
     'DEFAULT_FILTER_BACKENDS': (
-        'rest_framework.filters.DjangoObjectPermissionsFilter',
+        'resolwe.permissions.filters.ResolwePermissionsFilter',
```

Comment:
Interesting idea, I also don't like that currently the storage use case is hard-coded in the filter. So we should either generalize the filter (make the filter configurable from the viewset as I suggested in the comment above) or do as Miha suggests here.